Constitution.txt

```diff
@@ -1,490 +1,264 @@
+Article VIII (Article 8 - Mode of Amendment)
+The Congress, whenever two thirds of both Houses shall deem it necessary, shall propose Amendments to this Constitution, or, on the Application of the Legislatures of two thirds of the several States, shall call a Convention for proposing Amendments, which, in either Case, shall be valid to all Intents and Purposes, as Part of this Constitution, when ratified by the Legislatures of three fourths of the several States, or by Conventions in three fourths thereof, as the one or the other Mode of Ratification may be proposed by the Congress; Provided that no State shall be deprived of its equal Suffrage in the Senate, nor of proportional representation in the House of Representatives, and that this protection shall be preserved through any Amendment.
 
-Amendment XXVI (Amendment 26 - Reduction of Voting Age Qualification)
-1: The right of citizens of the United States, who are 18 years of age or older, to vote, shall not be denied or abridged by the United States or any state on account of age. affects 15
+Article IX (Article 9 - Prior Debts, National Supremacy, Oaths of Office)
+1: No Amendment to this constitution may free these United States from Debts or Treaties previously entered into, except by the terms of such Debts and Treaties.
 
-2: The Congress shall have the power to enforce this article by appropriate legislation.   ratified #26
+2: This Constitution, and the Laws of these United States which shall be made in Pursuance thereof; and all Treaties made, or which shall be made, under the Authority of these United States, shall be the supreme Law of the Land; and the Judges in every State shall be bound thereby, any Thing in the Constitution or Laws of any State to the Contrary notwithstanding.
 
-Amendment XXVII (Amendment 27 - Congressional Pay Limitation)
-No law varying the compensation for the services of the Senators and Representatives shall take effect until an election of Representatives shall have intervened.   ratified #27
+3: The Senators and Representatives before mentioned, and the Members of the several State Legislatures, and all executive and judicial Officers, both of these United States and of the several States, shall be bound by Oath or Affirmation, to support and defend this Constitution above all else; but no other religious Test shall ever be required as a Qualification to any Office or public Trust under these United States.
```

Comment:
The careful reader will note the addition of the word "other" regarding religious tests. This is because, in order to serve as a member of the Government of the Union, one must swear to be bound to the Declaration of Rights and Responsibilities, which is an article of faith.